src/encord_active/app/data_quality/sub_pages/summary.py

```diff
@@ -1,29 +1,148 @@
 
+from encord_active.app.common.components.data_quality_summary import summary_item
 from encord_active.app.common.components.metric_summary import render_metric_summary
 from encord_active.app.common.components.tags.tag_creator import tag_creator
 from encord_active.app.common.page import Page
 from encord_active.app.common.state import get_state
-from encord_active.lib.dataset.outliers import get_iqr_outliers
+from encord_active.lib.charts.data_quality_summary import (
+    create_image_size_distribution_chart,
+    create_outlier_distribution_chart,
+)
+from encord_active.lib.dataset.outliers import (
+    IqrOutliers,
+    MetricWithDistanceSchema,
+    OutlierStatus,
+    get_iqr_outliers,
+)
+from encord_active.lib.dataset.summary_utils import (
+    get_all_image_sizes,
+    get_median_value_of_2D_array,
+)
 from encord_active.lib.metrics.utils import (
+    MetricData,
     MetricScope,
     load_available_metrics,
     load_metric_dataframe,
 )
 
+_COLUMNS = MetricWithDistanceSchema
+
+
+class MetricOutlierInfo(NamedTuple):
+    metric: MetricData
+    df: pd.DataFrame
+    iqr_outliers: IqrOutliers
+
 
 class SummaryPage(Page):
     title = "📑 Summary"
+    _severe_outlier_color = "tomato"
+    _moderate_outlier_color = "orange"
+    _summary_item_background_color = "#fbfbfb"
 
     def sidebar_options(self):
         self.row_col_settings_in_sidebar()
         tag_creator()
 
     def build(self, metric_scope: MetricScope):
+
+        image_sizes = get_all_image_sizes(get_state().project_paths.project_dir)
+        median_dimension = get_median_value_of_2D_array(image_sizes)
+
+        metrics = load_available_metrics(get_state().project_paths.metrics, metric_scope)
+        total_severe_outliers = set()
+        total_moderate_outliers = set()
+
+        all_metrics_plotting = pd.DataFrame(columns=["metric", "total_severe_outliers", "total_moderate_outliers"])
+
+        metrics_info = []
+        for metric in metrics:
+            original_df = load_metric_dataframe(metric, normalize=False)
+            res = get_iqr_outliers(original_df)
+            if not res:
+                continue
+
+            df, iqr_outliers = res
+
+            all_metrics_plotting = pd.concat(
+                [
+                    all_metrics_plotting,
+                    pd.DataFrame(
+                        {
+                            "metric": [metric.name],
+                            "total_severe_outliers": [iqr_outliers.n_severe_outliers],
+                            "total_moderate_outliers": [iqr_outliers.n_moderate_outliers],
+                        }
+                    ),
+                ],
+                axis=0,
+            )
+
+            for _, row in df.iterrows():
+                if row[_COLUMNS.outliers_status] == OutlierStatus.severe.value:
+                    total_severe_outliers.add(row[_COLUMNS.identifier])
+                elif row[_COLUMNS.outliers_status] == OutlierStatus.moderate.value:
```

Comment:
[nitpick] 💅

when using string enums there is not need to compare string values.

```suggestion
                if row[_COLUMNS.outliers_status] == OutlierStatus.severe:
                    total_severe_outliers.add(row[_COLUMNS.identifier])
                elif row[_COLUMNS.outliers_status] == OutlierStatus.moderate:
```